book/logging-problems-and-solutions.md

```diff
@@ -63,3 +63,38 @@ the progress. In this case you need to change settings via application config fi
     ]
 ]
 ```
+
+Write different logs to different files
```

Comment:
Makes sense to have it. Probably `logging-problems-and-solutions.md` should be separated into individual small recipes.